src/containers/Viewer.jsx

```diff
@@ -0,0 +1,57 @@
+import styles from '../styles/viewer'
+
+import React from 'react'
+import { connect } from 'react-redux'
+import { withRouter } from 'react-router'
+
+import { STACK_FILES_DOWNLOAD_PATH } from '../constants/config'
+
+import ViewerToolbar from '../containers/ViewerToolbar'
+
+const Viewer = ({ current, previous, next, navigateToPhoto }) => {
+  return (<div className={styles['pho-viewer-wrapper']} role='viewer'>
+    <ViewerToolbar />
+    <div className={styles['pho-viewer-content']}>
+      <div className={styles['photo-viewer-nav-previous']} onClick={() => navigateToPhoto(previous)}>
```

Comment:
As I was talking on IRC about CSS prefixes, maybe we should be consistent and use `pho-viewer-nav-previous`here ?